corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -612,9 +612,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                                     markers.forEach(m => m.setIcon(locationIcon));
                                     marker.setIcon(selectedLocationIcon);
 
+                                    // Offset by breadcrumbs w/w/o small screen map
```

Comment:
what does w/w/o stand for?

Reply:
With or without. Can rephrase for clarity